static/js/portico/landing-page.js

```diff
@@ -178,6 +178,21 @@ var apps_events = function () {
 var events = function () {
     ScrollTo();
 
+    // Testimonial carousel height resizing
+    $(window).on('resize', function () {
+        // Before resizing, switch to the first element, because it
+        // holds the longest text, use its height as the min-height.
+        var activeTest = $("div.item.active.quote-container").removeClass("active");
+        $("div.quote-container:nth-child(1)").addClass("active");
+
+        $('#quote-carousel').css('min-height', '');
```

Comment:
Can you explain why it's necessary to reset the min-height to an empty string before resetting it?